### ISSUE TITLE
[BUG] Node.js version of `getCompliance(html, 'my-label')` does not mark violation `ignored: true` in newly generated report file

### ISSUE THREAD

**Issue by liunate** [IBMa/equal-access#861]
Node.js version of accessibility checker running in Jest.

Steps and test to reproduce this at jest without specifying any config:

```js
  const { report: actual } = await aChecker.getCompliance(
    '<html><header></header><body><div>123</div></body></html>',
    'nate'
  );

  const assertActual = aChecker.assertCompliance(actual);

  expect(assertActual).toBe(0);
```

1. Run attached test
2. Run `assertCompliance(report)` it returns `2` (this is the expected behavior as we don't have any baseline yet)
2. Copy the results file **/results/nate.json** to **/baselines/nate.json**
3. Run attached test again
4. New **/results/nate.json** still has all its violations `ignored: false` and `summary.count.ignored: 0` (expecting `true` and `ignored` bumped but it doesn't happen)
5. Run `assertCompliance(report)` it returns `0` (expected behavior as new scan find out exactly same violations as baseline)
